code/src/Components/AlbumImage.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const AlbumImage = (prop) => {
+    return (
+              <div className="image-container">    
+                  <img className="album-image"src={prop.album.images[1].url} alt="Album"/>
+                  <div className="icons-container">
+                    <div className="hover-container">
+                        <img className="heart" src="./icons/heart.svg" alt="heart icon"/>
+                        <a href={prop.album.external_urls.spotify} title="link to spotify" target="_blank" rel="noopener noreferrer">
+                            <img className="play" src="./icons/play.svg" alt="play button" />
+                        </a>
+                        <img className="dots" src="./icons/dots.svg" alt="dots icon" />
```

Comment:
Your icons do not always scale well. Sometimes they overlay the container. 
![image](https://user-images.githubusercontent.com/50926159/111841110-bb92de00-88fd-11eb-828b-13c0b3a30cc9.png)